src/abstract.js

```diff
@@ -793,16 +793,93 @@ Sk.abstr.sattr = function (obj, nameJS, data, canSuspend) {
 };
 goog.exportSymbol("Sk.abstr.sattr", Sk.abstr.sattr);
 
-Sk.abstr.iter = function (obj) {
+
+Sk.abstr.iternext = function (it, canSuspend) {
+    return it.tp$iternext(canSuspend);
+};
+goog.exportSymbol("Sk.abstr.iternext", Sk.abstr.iternext);
+
+
+/**
+ * @constructor
+ */
+function seqIter(obj) {
```

Comment:
This is a bit weird a global function just sitting around here couldn't this be added to Sk.abstr also?